software/lha.yml

```diff
@@ -0,0 +1,10 @@
+name: LHA
+website_url: https://github.com/javalikescript/lha
+description: Light Home Automation application. LHA is fully extensible using Blockly, HTML or Lua. It includes extensions such as ConBee, Philips Hue or Z-Wave JS.
+licenses:
+  - MIT
+platforms:
+  - Lua
+tags:
+  - Internet of Things (IoT)
+source_code_url: https://github.com/javalikescript/lha
```

Comment:
source_code_url should be the line under website_url.

Reply:
As its a yaml file, the order doesn't matter too much.

Reply:
Moved in commit 941ced6c13e941bc84e70e096604f4018d79baa0